src/devices/Bmi088/AccelerometerOutputDataRate.cs

```diff
@@ -1,32 +1,32 @@
-﻿namespace Iot.Device.GroveBmi088Device
+﻿namespace Iot.Device.Bmi088Device
 {
     /// <summary>
     /// Data rate for the accelerometer
     /// </summary>
     public enum AccelerometerOutputDataRate : byte
     {
         /// <summary>See datasheet</summary>
-        ODR_12 = 0x05,
+        Odr12 = 0x05,
```

Comment:
I think @Ellerbach meant to also expand this, also make sure to make it as similar as possible to other APIs:
https://github.com/dotnet/iot/blob/master/src/devices/Ads1115/DataRate.cs
https://github.com/dotnet/iot/blob/master/src/devices/Nrf24l01/DataRate.cs

also make sure units are clear (either included in the name or xml doc)